bot/resources/evergreen/trivia_quiz.json

```
@@ -269,5 +269,160 @@
       "answer": "1492",
       "info": "The explorer Christopher Columbus made four trips across the Atlantic Ocean from Spain: in 1492, 1493, 1498 and 1502. He was determined to find a direct water route west from Europe to Asia, but he never did. Instead, he stumbled upon the Americas"
     }
+  ],
+  "math": [
+    {
+      "id": 201,
+      "question": "What is the highest power of a biquadratic polynomial",
+      "answer": "4"
+    },
+    {
+      "id": 202,
+      "question": "What is the formula for surface area of a sphere",
+      "answer": [
+        "4pir^2",
+        "4πr^2"
+      ]
+    },
+    {
+      "id": 203,
+      "question": 203,
+      "answer": 203
+    },
+    {
+      "id": 204,
+      "question": 204,
+      "answer": 204
+    },
+    {
+      "id": 205,
+      "question": 205,
+      "answer": 205
+    },
+    {
+      "id": 206,
+      "question": 206,
+      "answer": 206
+    },
+    {
+      "id": 207,
+      "question": 207,
+      "answer": 207
+    },
+    {
+      "id": 208,
+      "question": 208,
+      "answer": 208
+    },
+    {
+      "id": 209,
+      "question": 209,
+      "answer": 209
+    },
+    {
+      "id": 210,
+      "question": 210
+      "answer": 210
```

Smh, bad reviewing, this has a bug
```suggestion
      "question": 210,
      "answer": 210
```